zgw/object-management/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+  com.ritense.documentenapi.DocumentenApiAutoConfiguration
```

Comment:
This autoconfiguration is not correct. This new module doesnt have a autoconfiguration yet. Either:
- create empty autocofiguration in this module. or
- make this file empty for now